x-pack/platform/plugins/shared/ml/public/application/supplied_configurations/supplied_configurations_flyout/flyout.tsx

```diff
@@ -115,12 +116,14 @@ export const SuppliedConfigurationsFlyout: FC<Props> = ({ module, onClose }) =>
     [tabs, selectedTabId]
   );
 
+  const flyoutTitleId = useGeneratedHtmlId();
+
   return (
     <EuiFlyout
       size="l"
       ownFocus
       onClose={onClose}
-      aria-labelledby={'supplied-configurations-flyout'}
+      aria-labelledby={flyoutTitleId}
```

Comment:
I confirmed that the dialog title is now being set in the Accessibility Tree View in Chrome Dev tools.

But just to confirm, are we expecting the title to be read out in MacOS VoiceOver, as it just reads out the generic 'You are in a modal dialog' currently? I think I checked this before, and believe it might be a limitation of VoiceOver? Could you confirm the expected behavior here please @alexwizp 

<img width="677" height="170" alt="Screenshot 2025-08-05 at 15 22 36" src="https://github.com/user-attachments/assets/a4b8c3e3-b21d-4aa5-94c4-8bbdeb525cea" />

Reply:
Thanks for the clarification @alexwizp . That's what I suspected. All looks good in the accessibility tree.